gearcmd/worker_test.go

```diff
@@ -24,6 +26,24 @@ func getSuccessResponseWithConfig(payload string, config TaskConfig, t *testing.
 	return string(mockJob.OutData())
 }
 
+func testRetryOnFailure(retryCount int) ([]byte, error) {
+	// Get a temp file for the script to hold its state.
+	file, err := ioutil.TempFile("", "temp")
+	if err != nil {
+		return nil, fmt.Errorf("Could not create temporary file: %s", err.Error())
+	}
+	filename := file.Name()
+	defer os.Remove(filename)
+	defer file.Close()
+	mockJob := mock.CreateMockJob(filename)
```

Comment:
The bash script uses it to store the number of times it has been run.  So after the first run, it'll hold 1, then after the next, 2.  Etc.